### PR TITLE
PB-1152: Support preparsed ecs realtime logs

### PR DESCRIPTION
**Pull request by ltflb-bgdi**
Realtime logs are now ecs pre-parsed by firehose lambda function. This saves performance and improves testability.

### CODE REVIEW

File: packages/ppbgdi/data_stream/cloudfront_logs/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -31,573 +31,28 @@ processors:
...
+      description: Process cloudfront realtime logs
+      if: ctx.first_char == '{'
+      name: '{{ IngestPipeline "pipeline_realtime_logs" }}'
+      tag: pipeline_realtime_logs
+      ignore_missing_pipeline: false
+  - pipeline:
+      description: Process cloudfront standard logs
+      if: ctx.first_char != '{'
+      name: '{{ IngestPipeline "pipeline_standard_logs" }}'
+      tag: pipeline_realtime_logs
```
Comment (boecklic) [inline packages/ppbgdi/data_stream/cloudfront_logs/elasticsearch/ingest_pipeline/default.yml:50]: should probably be `pipeline_standard_logs`?

File: packages/ppbgdi/data_stream/cloudfront_logs/fields/fields.yml [hunk 2]
```diff
@@ -79,8 +79,7 @@
       type: keyword
       description: |-
         Kinesis type.
-- name: http.request.headers.*
+- name: http.request.header.*
   type: keyword
   description: Not yet but already in use ecs fields
```
Comment (boecklic) [inline packages/ppbgdi/data_stream/cloudfront_logs/fields/fields.yml:84]: `Not yet *defined/approved* but already...`?

**Review by boecklic — APPROVED** [review]

File: packages/ppbgdi/data_stream/cloudfront_logs/fields/fields.yml (hunk 2 above)
Comment (ltflb-bgdi) [reply packages/ppbgdi/data_stream/cloudfront_logs/fields/fields.yml:84]: Done

File: packages/ppbgdi/data_stream/cloudfront_logs/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (ltflb-bgdi) [reply packages/ppbgdi/data_stream/cloudfront_logs/elasticsearch/ingest_pipeline/default.yml]: Thanks

**Pull request merged**